shell/bash.go

```diff
@@ -9,17 +11,24 @@ import (
 )
 
 var bashShellHooks = `
-if test -n "${PROMPT_COMMAND+_}"; then 
+if test -n "${PROMPT_COMMAND+_}"; then
   PROMPT_COMMAND="change_hermit_env; $PROMPT_COMMAND"
 else
   PROMPT_COMMAND="change_hermit_env"
 fi
 
-complete -o nospace -C "$HOME/bin/hermit" hermit
+complete -o nospace -C "%s" hermit
```

Comment:
Sorry to be a pain, but it only just occurred to me that the Go changes here are probably not necessary. Instead it would be simpler to just have `${HERMIT_ROOT_BIN:-$HOME/bin/hermit}` in the two shell scripts.

Reply:
Yep!